Backend/dev/migrations/dynamic-offer-driver-app/0500-delivery.sql

```diff
@@ -108,3 +108,11 @@ VALUES
 --     null,
 --     null
 --     from atlas_driver_offer_bpp.merchant_operating_city as m where merchant_id = '96dd7f78-787e-4a0b-8675-e9e6fe93bb8f');
+
+
+
+-- for making bike not as default --
+update atlas_driver_offer_bpp.vehicle_service_tier
+set default_for_vehicle_variant = '{}',
+    auto_selected_vehicle_variant = '{BIKE,DELIVERY_BIKE}'
+where service_tier_type = 'BIKE';
```

Comment:
City can be only Kolkata in Where clause as now Bike we are going Live in Hyderabad and other cities too so there we will not have Delivery Bike.

Reply:
Also, why are we `default_for_vehicle_variant = '{}'` doing this ? We can have BIKE here ?

Reply:
😢 
I also want BIKE to be default and we always keep same tier as default in NY,
but qa and pm both wants that driver should be able to unselect BIKE stier also , if he only wants to do parcel delivery.